templates/http/base/deployment-model-server.yaml

```diff
@@ -26,7 +26,7 @@ spec:
       initContainers:
       - name: model-file
         image: {{values.initContainer}}
-        command: ['/usr/bin/install', "/model/model.file", "/shared/"]
+        command: {{values.command}}
```

Comment:
`values.modelCommand` in case we need to specify the command for the deployment in the future?

Reply:
updated